src/Application/OpcUa/AlarmConversion.cs

```diff
@@ -0,0 +1,197 @@
+﻿// SPDX-License-Identifier: MIT. 
+// Copyright Contributors to the Open Manufacturing Platform.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using AutoMapper;
+using OMP.Connector.Domain;
+using OMP.Connector.Domain.Models.OpcUa.Attributes;
+using OMP.Connector.Domain.OpcUa;
+using OMP.Connector.Domain.Schema.Interfaces;
+using OMP.Connector.Domain.Schema.SensorTelemetry;
+using OMP.Connector.Domain.Schema.SensorTelemetry.PrimitiveTypes.Base;
+using Opc.Ua;
+
+namespace OMP.Connector.Application.OpcUa
+{
+    internal static class AlarmConversion
+    {
+        private const string ArrayPostfix = "[]";
+
+        #region Conversion methods
+
+        internal static List<object> ConvertToAlarm(object value, IMapper mapper)
+        {
+            //if (IsPrimitiveStringOrNull(value))
```

Comment:
Can we delete all the dead code in here?

Reply:
Done